src/main/kotlin/ink/abb/pogo/scraper/Settings.kt

```diff
@@ -93,6 +94,17 @@ class Settings(val properties: Properties) {
         listOf()
     }
 
+    val autoEvolve = getPropertyIfSet("list of pokemon you want to evolve when able to", "auto_evolve", "CATERPIE,PIDGEY,WEEDLE", String::toString).split(",")
+
+    //This method only exists because I wasn't sure if this data was stored somewhere else. If it is then this can be removed.
+    private fun getCandyByPokemon(): Map<Int, Int> {
+            val lines = File("pokemon-candy.csv").readLines()
```

Comment:
@xenxes : do you have a link to a compiled jar of this to check out? I can't compile because windows is stupid and I get too long of a file path when compiling dependencies.